numa.c

```diff
@@ -78,8 +78,8 @@ static void add_one_node(const char *nodename)
 			cpus_clear(new->mask);
 		} else {
 			cpumask_parse_user(cpustr, ret, new->mask);
-			free(cpustr);
 		}
```

Comment:
The movement of the free call looks good, but now that the else clause is now a single statement, could you please eliminate the braces  to keep it in line with the rest of the file style please?  In fact, can you fix the if clause too while you're at it?  I should have done that one long ago.

Reply:
I have modify the patch as your suggest, thanks.